staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -881,8 +881,8 @@ func NewDefaultAPIGroupInfo(group string, scheme *runtime.Scheme, parameterCodec
 }
 
 // getOpenAPIModels is a private method for getting the OpenAPI models
-func (s *GenericAPIServer) getOpenAPIModels(apiPrefix string, apiGroupInfos ...*APIGroupInfo) (*spec.Swagger, error) {
-	if s.openAPIConfig == nil {
+func (s *GenericAPIServer) getOpenAPIModels(apiPrefix string, apiGroupInfos ...*APIGroupInfo) (map[string]*spec.Schema, error) {
+	if s.openAPIV3Config == nil {
```

Comment:
What are the consequences of this change? is the openAPIV3Config always set when openapiConfig is set?

Reply:
People who don't have `openAPIV3Config` set will be disabling SSA too. Until https://github.com/kubernetes/kubernetes/pull/114998 lands and then they will get an error on startup

Reply:
Thanks, sounds like David was on board with that. You could add a comment/TODO here that we need #114998.

Reply:
added